subaccounts.go

```diff
@@ -44,30 +49,30 @@ func (c *Client) SubaccountCreate(s *Subaccount) (res *Response, err error) {
 func (c *Client) SubaccountCreateContext(ctx context.Context, s *Subaccount) (res *Response, err error) {
 	// enforce required parameters
 	if s == nil {
-		err = fmt.Errorf("Create called with nil Subaccount")
+		err = errors.New("Create called with nil Subaccount")
 	} else if s.Name == "" {
-		err = fmt.Errorf("Subaccount requires a non-empty Name")
+		err = errors.New("Subaccount requires a non-empty Name")
 	} else if s.KeyLabel == "" {
-		err = fmt.Errorf("Subaccount requires a non-empty Key Label")
+		err = errors.New("Subaccount requires a non-empty Key Label")
 	} else
 	// enforce max lengths
 	if len(s.Name) > 1024 {
-		err = fmt.Errorf("Subaccount name may not be longer than 1024 bytes")
+		err = errors.New("Subaccount name may not be longer than 1024 bytes")
 	} else if len(s.KeyLabel) > 1024 {
-		err = fmt.Errorf("Subaccount key label may not be longer than 1024 bytes")
+		err = errors.New("Subaccount key label may not be longer than 1024 bytes")
+	} else if s.IPPool != "" && len(s.IPPool) > 20 {
+		err = errors.New("Subaccount ip pool may not be longer than 20 bytes")
 	}
 	if err != nil {
 		return
 	}
 
 	if len(s.Grants) == 0 {
-		s.Grants = availableGrants
+		s.Grants = SubaccountGrants
```

Comment:
This might be an important rule to document in the comments. Not sure a user will expect all permissions if none were passed in.